data/free worlds checkmate.txt

```diff
@@ -2410,96 +2410,122 @@ mission "FWC Terrorists 1A"
 			`	Korban looks down, "Understandable," then looks back up at you with a smile, "but such is the cost of business, we say.`
 			label absolutely
 			`	"Now, we know that the terrorists still have nuclear weapons. We sent a ship to scout out their strength that barely made it out alive. We have a plan to take care of that, though.`
-			`	"We've found ourselves in possession of a strange device. It took us a long time to even figure out how to activate it. But when we did, we discovered that it's a cloaking device. It obscures a ship not just from the visible spectrum, but from radar as well, and every other sensor we've tested except for gravitational ones. I'm certain it would hide you from the tracking systems on the nukes."`
-			`	Freya says, "But it would only hide one ship. That doesn't sound like it would do much good."`
-			`	Korban says, "They have a limited number of nukes. Our plan was to send a ship in cloaked, have it uncloak long enough to draw their fire, and then cloak to avoid the missiles. Basically, see how many missiles we can get them to spend before the main attack fleet arrives."`
+			`	"Before the war, we began to work heavily toward developing automation for use on ships. You may have bought and used one of our Vanguards. Such a small crew needed to man a ship of such a size. We made some breakthroughs during the war that allowed us to drastically reduce the crew needed to pilot ships of any size, and we'll be sending a fleet of such ships as cannon fodder to have these terrorists waste their nukes."`
+			`	Freya looks shocked, "You're sending people on a suicide mission to take the nukes out of play? Can't we use diplomacy to stop adding bodies to the pile that this war has created?"`
+			`	Korban says, "These are terrorists, dear. They aren't to be negotiated with. But fear not, because these ships won't be manned. They'll be fully autonamous warships. Their combat skills are rough around the edges, and this will be an expensive loss - each of these ships costs us at least tens times the production of a normal ship - but much cheaper than the loss of innocent human lives."`
```

Comment:
"cheaper than the loss of innocent human lives" doesn't feel like a syndicate thing to say, but maybe having the navy breathing down their necks has something to do with it.

Reply:
He's simply appealing to the emotions of Freya and the player. You don't get to the position of the CEO of the largest corporation in the galaxy without knowing how to do a good sales pitch.